external-task-worker/src/main/java/info/novatec/micronaut/camunda/externaltask/worker/TopicSubscriber.java

```diff
@@ -0,0 +1,37 @@
+package info.novatec.micronaut.camunda.externaltask.worker;
+
+import io.micronaut.context.event.ApplicationEventListener;
+import io.micronaut.runtime.server.event.ServerStartupEvent;
+import org.camunda.bpm.client.ExternalTaskClient;
+import org.camunda.bpm.client.task.ExternalTaskHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Singleton;
+
+@Singleton
+public class TopicSubscriber implements ApplicationEventListener<ServerStartupEvent> {
+
+    private static final Logger log = LoggerFactory.getLogger(TopicSubscriber.class);
+
+    protected ExternalTaskHandler externalTaskHandler;
+    protected Configuration configuration;
+    protected ExternalTaskSubscription externalTaskSubscriptionAnnotation;
+
+    TopicSubscriber(ExternalTaskHandler externalTaskHandler, Configuration configuration) {
+        this.externalTaskHandler = externalTaskHandler;
+        this.configuration = configuration;
+        this.externalTaskSubscriptionAnnotation = externalTaskHandler.getClass().getAnnotation(ExternalTaskSubscription.class);
```

Comment:
This looks as if we can only have one implementation of an "ExternalTaskSubscription". However, I should be able to define multiple ones each subscribing on a different topic.